README.md

```diff
@@ -1,13 +1,53 @@
-## supreme-octo-fishstick
-
-### Project
-This project includes a frontend (React) and backend (Flask) to serve as a website for querying a database of arithmetic dynamical systems. Currently backend only uses static data but will eventually communicate with a database (PostgreSQL)
- 
-### Dependencies
-* Python dependencies: flask, flask_cors
-* Frontend: Node, React, MUI 
-### How to run backend
-* run ‘python server.py’ inside backend folder
-### How to run frontend
-* run 'npm install' for first time 
-* run ‘npm run start’ inside frontend folder, website should open up automatically in browser
+## DADS: A Database of Arithmetical Dynamic Systems
+DADS is a web application designed for researchers to access information about a vast array of arithmetical dynamical systems that would otherwise take a long time to compute independently. This project was developed by the SLU Capstone Project team under Open Source with SLU in collaboration with Dr. Benjamin Hutz, a professor of mathematics and statistics at Saint Louis University.
+
+### Features
+* Home Page: A brief description of the application and a link to the GitHub repository.
+* Explore Systems Page: Allows users to filter and query through the database for various properties like automorphism group cardinality and degree. The results are displayed in a paginated data table with a statistics summary to the right which shows metrics about the returned systems, such as average height.
+* System Details Page: Each system in the paginated table has a label which can be selected to show a page with additional information about the system.
+
+### Tech Stack
+* Frontend: React
+* Backend: Python
+* Database: PostgreSQL
+
+### Getting Started
+* Clone the repository from GitHub.
+* Install the necessary dependencies for React and Python using PIP and npm install 
+* Set up the PostgreSQL database.
+* Run the application.
+* For detailed instructions, please refer to the Installation Guide.
+
+### Contributing
+We welcome contributions from the community. Please refer to the Contributing Guide for more information.
```

Comment:
Could you please add an empty file for the contributor's guide and give the hyperlink here? I will work on the contributor's guide.